### PR TITLE
Ch6. Adjust arguments for differential learning rates in Adam optimizer

### PR DESCRIPTION
**Pull request by MarcusFra**
Hi Ian,

In Chapter 6 the differential learning rates are given by a list as an argument of the Adam optimizer. 

I couldn't find any support on a list type, thus I defined the learning rates for every layer explicitly. If you support this implementation, please also check whether I got you right which learning rates are supposed for the respective layers. (from the book: "we're also going to use differential learning rates of `[1e-2,1e-4,1e-8]`, with the highest learning rate applied to our classifier (as it requires the most training!) and slower rates for the already-trained backbone.") - I'm not sure whether I selected the layers right, and also whether still to change 1e-2 with 1e-8 (the argument's position), so the last layers would have the higher rate - but, I thought since the list was given in decreasing order, I first try to implement it that way.

### CODE REVIEW

**Comment by MarcusFra** [comment]
descending, I meant ;)

**Review by falloutdurham — APPROVED** [review]
